routes/web.php

```diff
@@ -77,7 +77,7 @@
 Route::get('/praktikan', function () {
     $user = Auth::guard('praktikan')->user();
     $user->kelas = Kelas::where('id', $user->kelas_id)->first()->kelas;
-    $allAsisten = Asisten::all();
+    $allAsisten = Asisten::orderBy('kode','asc')->get();
```

Comment:
Liking this small sorting can change everything XD

Reply:
yepp that is correct tho haha

Reply:
hahaha that's right